docs/docs/rest-api/public/api/v2/types/pod-status.raml

```diff
@@ -0,0 +1,58 @@
+#%RAML 1.0 DataType
+type: object
+description: |
+  A pod allows one to launch a collection co-located (on the same agent) containers
+  that share the same network namespace and that may share the same volumes.
+  Resources are specified on a per-container basis.
+additionalProperties: false
+properties:
+  id:
+    type: !include pathId.raml
+    description:|
+      Fully qualified name of the pod that this status object is describing.
+  spec:
+    type: !include pod.raml
+    description:|
+      The latest version of the pod specification (that has the same pod ID).
+  status:
+    type: string
+    enum: [ DEGRADED, STABLE, TERMINAL ]
```

Comment:
I tried to capture that in DEGRADED by referencing "stable pod instances" in the description. Or do you think this overloads the meaning of DEGRADED?

I also don't know what @unterstein and @meichstedt have cooked up for instance states. could influence what we have here